app/src/main/java/com/ubclaunchpad/room8/adapter/PendingInvAdapter.java

```diff
@@ -25,15 +26,16 @@ public PendingInvAdapter(Map<String,String> pendingInvites) {
     }
 
     public class ViewHolder extends RecyclerView.ViewHolder {
-        public TextView groupNameTextView;
-
+        TextView groupNameTextView;
+        Button acceptBtn;
```

Comment:
Im trying to implement a new style convention. If your member variable is a component in the layout, name it something like "btnAccept" for buttons.